src/main/java/hu/wurfel/new_school_reference/student/StudentService.java

```diff
@@ -28,16 +28,16 @@ public Student toEntity(StudentDto dto) {
 	}
 
 
-	public List<Student> findAllByName (String name){
-		return repo.findAllByNameAndDeletedIsFalse(name);
+	public List<Student> findAllByName (String name, boolean deleted){
```

Comment:
```suggestion
	public List<Student> findAllByNameAndDeleted(String name, boolean deleted){
```
Nem 100% must, de ha már 2 keresési paraméter van és az egyik (teljesen jól) jelezve van a service method nevében és mindkettő a repo method nevében (nyilván), akkor a service method neve is egészüljön ki.

Reply:
Ezt nyilván minden ilyenre érvényes.

Reply:
done
ű